soap/soap.d.ts

```diff
@@ -8,13 +8,19 @@
 declare module 'soap' {
     import * as events from 'events';
 
-    class WSSecurity {
+    class Security {
```

Comment:
There is no value called `Security` exposed on the module at runtime. this class gives you nothing in terms of value since it is empty and Typescript has a structural type system. please remove it.